api/PerformanceObserverEntryList.json

```diff
@@ -17,9 +17,22 @@
           "ie": {
             "version_added": false
           },
-          "nodejs": {
-            "version_added": "8.5.0"
-          },
+          "nodejs": [
+            {
+              "version_added": "19.0.0"
+            },
+            {
+              "version_added": "16.7.0",
+              "partial_implementation": true,
+              "notes": "Available as a part of the <code>perf_hooks</code> module."
+            },
+            {
+              "version_added": "8.5.0",
+              "version_removed": "16.7.0",
+              "partial_implementation": true,
+              "notes": "Only available as an internal instance."
+            }
```

Comment:
If it isn't exposed to the user, I don't think we should keep this entry.

Reply:
the interface is simply exposed directly, but still can be accessed in other way

```js
const obs = new perf_hooks.PerformanceObserver((list, observer) => {
  console.log(list.__proto__.constructor); // same as `PerformanceObserverEntryList`
});
obs.observe({ entryTypes: ['mark'], buffered: true });
perf_hooks.performance.mark('test'); 
```

and there is a rule that parent features can't have higher version than children features, so this item is necessary here